tests/test_regression.py

```diff
@@ -7,14 +7,66 @@
 from autocorpus.configs.default_config import DefaultConfig
 
 
-def test_autocorpus(data_path: Path):
-    """A regression test for the main Auto-CORPus class on the Auto-CORPus paper.
+def test_pmc_autocorpus():
+    """A regression test for the main autoCORPus class, using the current PMC config on the AutoCORPus Paper.
+
+    The test data output is built using `run_app.py` with the following arguments:
```

Comment:
This should be updated
```suggestion
    The test data output is built by running Auto-CORPus from the root of the repo with the
    following arguments
```